include/maxscale/buffer.h

```diff
@@ -145,6 +147,7 @@ typedef struct gwbuf
     gwbuf_type_t    gwbuf_type; /*< buffer's data type information */
     HINT            *hint;  /*< Hint data for this buffer */
     BUF_PROPERTY    *properties; /*< Buffer properties */
+    struct server   *server;
```

Comment:
A small inline comment about what is struct is would be good. It helps when the documentation for member variables gives a small hint as to what it's going to be used for.